src/core/font_substitutions.js

```diff
@@ -400,8 +400,6 @@ function getFontSubstitution(
   baseFontName,
   standardFontName
 ) {
-  let mustAddBaseFont = false;
-
```

Comment:
*Please note:* Moving this down is unrelated to the rest of the patch, however I happened to notice that initializing this variable is unnecessary for already *cached* standard-fonts.